ansible_navigator/actions/run.py

```diff
@@ -530,12 +538,18 @@ def _take_step(self) -> None:
     def _run_runner(self) -> None:
         """spin up runner"""
         executable_cmd: Optional[str]
+
+        if self.mode == "stdout_w_artifact":
```

Comment:
The if block can be moved within `def mode` property. All the mode related setting can be handled in one place by doing so.

Reply:
The reason it's here, is that def mode needs to return 1 of 3 modes:

1 = interactive
2 = stdout_w_artificat
3 = stdout

The second is needed within handle message to determine if the stdout mesaages should be printed